.github/workflows/python_tests.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       - name: Install tox
         run: pip install tox==3.27.1
       - name: Run tests basic unix
-        if: contains(matrix.os, 'ubuntu') || startsWith(matrix.os, 'macos')
+        if: contains(matrix.os, 'ubuntu-20.04') || startsWith(matrix.os, 'macos')
```

Comment:
Do we need this change? I actually like just gating on ubuntu, it seems cleaner since we won't need to update it if we change ubuntu versions

Reply:
I've tested it with _ubuntu_, but since now the matrix.os value is an array instead of a single string, the _contains_ function doesn't trigger the test, apparently when working with arrays the function doesn't check for substrings,  only _ubuntu-20.04_ is working.